packages/tc-ui/src/pages/view/template.jsx

```diff
@@ -19,7 +19,7 @@ const Properties = ({ fields, data, assignComponent }) => {
 		)
 		.map(([name, item]) => {
 			const viewModel = {
-				value: data[name],
+				value: data[name] || data[`${name}_rel`],
```

Comment:
This will be vulnerable to e.g. if the value is `false`, then it will end up looking for `_rel` and displaying undefined

A more robust approach - and easier to read the intent I think, would be `\`${name}_rel\` in data ?  data[`${name}_rel`] : data[name]`

Even that's not perfect as the check we really want is 'is `name` the name of a relationship property, in that case use _rel, otherwise just use name', so a condition which checks for the relationshipness of `name` would be ideal